R_D_HW_22/Select 2

```diff
@@ -0,0 +1,2 @@
+
+SELECT  users.id, users.first_name, users.last_name, books.title FROM users, books JOIN purchase on purchase.book_id = books.id ORDER BY users.id
```

Comment:
This query selects all books for all users. You need to join 2 tables here.
`select ... from purchase join users on purchase.user_id = users.id join books on purchase.book_id = books.id ...`